.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+example/dist/*
```

Comment:
I don't understand why this is needed 🤔

Reply:
didn't you say not to commit the example/dist/* files?

Reply:
Yeah, but I think it is already gitignored in example's gitignore

Reply:
Plus I don't think we have example folder? I guess we have example-preact, example-react and example-vanilla?